tests/features/front-end/list-view/view-layout.feature

```diff
@@ -2,33 +2,61 @@ Feature: layout and visual elements of an event in List View
 
 Background:
     Given I am a Visitor on a site with TEC
-    Given List View is enabled under Events --> Settings --> Display
-    Given the site timezone is America/Los_Angeles
-    Given "Number of events to show per page" is set to 10
+    And List View is enabled under Events --> Settings --> Display
+    And the site timezone is America/Los_Angeles
+    And "Number of events to show per page" is set to 10
 
 Scenario: display of single-day timed event in List View
+    Given I have a single-day timed event
```

Comment:
Always define events in context of current time: 
```
Given I have the following event:
| title  | start_date         | timezone                     |
| Test | tomorrow 9am | America/Los_Angeles |
```

This is still an automated test that requires translation into code, the structure above is still readable and clear but lends itself to be reused:
```
Given I have the following events:
| title     | start_date           | timezone                      | duration |
| Test 1 | tomorrow 9am    | America/Los_Angeles | 2H          |
| Test 2 | tomorrow 12pm | Europe/Paris                 | 5H          |
```

The same step, once standardized, can cover a lot of mileage. The columns would be the ones supported by the ORM syntax.

Reply:
I've been using something like this:
```
    Given there are 12 events on this site
        | event name       | start date-time        | duration       |
        | strawberry S-T-P | today -3 days -2 hours | 4 hours        |
        | banana M-A-P     | today -2 days all day  | 1 day          |
        | eggplant M-A-N   | today -2 days all day  | 4 days         |
```